pyscriptjs/src/components/pyscript.ts

```diff
@@ -9,6 +9,14 @@ const logger = getLogger('py-script');
 export function make_PyScript(runtime: Runtime) {
     class PyScript extends HTMLElement {
         async connectedCallback() {
+            if (this.hasAttribute('output')) {
+                const deprecationMessage = (
+                    "The 'output' attribute is deprecated. You should use the " +
+                    "'display' API to output the content to a specific element. " +
+                    'For example display(myElement, target="divID").'
```

Comment:
```suggestion
                    "The 'output' attribute is deprecated and ignored. You should use " +
                    "'display()' to output the content to a specific element. " +
                    'For example display(myElement, target="divID").'
```

Reply:
a bit of nitpick, but I think it's good to underline that it's not only deprecated, it's completely ignored